driver/compenv.ml

```diff
@@ -209,6 +209,17 @@ let read_OCAMLPARAM ppf position =
                                         "non-integer parameter for \"inline\""))
         end
 
+      (* color output *)
+      | "color" ->
+          begin try
+            let b = bool_of_string v in
```

Comment:
There already is a standardized interface for boolean parameters, which is to use `0`/`1` (see [Compenv.setter](https://github.com/ocaml/ocaml/blob/trunk/driver/compenv.ml#L127-L138)) rather than `bool_of_string`. You should feel free to generalize `Compenv.setter` to support calling an arbitrary function (as an auxiliary function).